translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,107 @@
+* Realm : क्षेत्र , Realm translates to क्षेत्र but re-translating back to 
+English it becomes *area* or *regoin*. This means that switching between
+ realms is lke talking to different people in different areas, which is 
+ how reals work in Zulip. , किसी अन्य क्षेत्र में जाएं
+* Stream : स्ट्रीम , Transliteration is used here as there is no way we can 
+translate stream and get the exact same meaning as in the context of Zulip. 
+धारा, प्रवाह, नदी, नाला, etc all represent different forms of flowing water and
+ therefore cannot be used. , केवल <stream> से संदेश दिखाएं
+* Topic : विषय , विषय means subject, but the other translations - प्रसंग means
+ context and प्रकरण means situation. Therefore विषय is used. , एक नए विषय को 
+ खोलें
+* Private/Invite-Only Stream : निजी / केवल आमंत्रण स्ट्रीम , Literal 
+translation is used here, therefore there will be no confusion on any side
+ while this word is used. , अपने निजी / केवल आमंत्रण स्ट्रीम देखें
+* Public Stream : सार्वजनिक स्ट्रीम , सार्वजनिक literally means "For all 
+people", and public streams are also for all users. Therefore this translation
+ is used. Other translations which are rejected are: प्रसिद्ध(famous), 
+ सामान्य(common), and प्रचलित(prevalent) , सार्वजनिक स्ट्रीम देखें
+* Bot : बॉट , Transliteration used here. Translation is available for "Robot"
+ instead, but that translation literally means a humanoid, whereas bots in
+ Zulip are just programs. , अपने बॉट देखें
+* Integration : एकीकरण , एकीकरण to any average Hindi user would mean something
+ like unification, and since it is used in Zulip, it would mean unifying 
+ Zulip with something. , फेसबुक के साथ जुलिप एकीकरण
+* Notification : अधिसूचना , अधिसूचना is the easiest to understand translation 
+of "Notification" and it literally means "Before information/Pre-information",
+ which is basically what notifications are: Alerting the user about a piece of
+ information waiting for them. , अधिसूचनाएं बंद करो
+* Alert Word : सतर्क शब्द , चेतावनी,सावधान,भय which are translations of alert 
+are all related to being scared and as a result alert. Therefore सतर्क is used
+ as a translation of alert. , अपने सतर्क शब्द देखें
+* View : देखें , One translation of view is राय, but that means opinion. 
+Therefore the formal translation of see is used here , बिना पढ़े हुए पोस्ट देखें
+* Home : मुख्य पृष्ठ , Translation of homepage is used here with a slight 
+modification. मुख पृष्ठ(home page) literally means face page or front page,
+ usually associated with books and magazines. Therefore मुख्य पृष्ठ is used
+ which means "Main page" , मुख्य पृष्ठ पर जाएं
+* Emoji : इमोजी , Transliteration used here, as there are no possible 
+translations for "emoji". Also the common Hindi folk is familiar with the 
+English word and therefore using transliteration shouldn't be a problem , इमोजी
+ शैली बदलें
+## Phrases (वाक्यांश)
+* Subscribe to a stream/Unsubscribe from a stream : किसी स्ट्रीम की सदस्यता लें

```

Comment:
Sorry, I don't have a Devnagiri input method here. What do you feel about *dhara ke sadasya bane*? That feels slightly less formal.

Reply:
I'm confused about this one because it would mean choosing between "Become a member of the stream" and "Take membership of a stream". Even I feel "धारा (स्ट्रीम) को सदस्य बनें" sounds less formal